bucket/wasmer.json

```
@@ -8,24 +8,22 @@
     },
     "architecture": {
         "64bit": {
-            "url": "https://github.com/wasmerio/wasmer/releases/download/v4.3.5/wasmer-windows.exe",
-            "hash": "7ebc26c5f845ab5a5ea9085418a95b282385d098b0f8b825abe9b9f3eca7a61c"
+            "url": "https://github.com/wasmerio/wasmer/releases/download/v4.3.5/wasmer-windows-amd64.tar.gz",
+            "hash": "e1acb2367b8d42736f4cb62fa405cd64dd55ad7844acf92231fe4454ba834976"
         }
     },
     "env_set": {
         "WASMER_DIR": "$dir",
         "WASMER_CACHE_DIR": "$dir\\cache"
     },
     "bin": "bin\\wasmer.exe",
-    "innosetup": true,
     "checkver": {
-        "github": "https://github.com/wasmerio/wasmer",
-        "regex": "tag/([\\w.-]+)"
+        "github": "https://github.com/wasmerio/wasmer"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/wasmerio/wasmer/releases/download/$version/wasmer-windows.exe"
+                "url": "https://github.com/wasmerio/wasmer/releases/download/$version/wasmer-windows-amd64.tar.gz"
```

I apologize for bringing this up late, but I have another important suggestion that,

```suggestion
                "url": "https://github.com/wasmerio/wasmer/releases/download/v$version/wasmer-windows-amd64.tar.gz"
```
We've fixed it to recognize version information without `v`, but we should include `v` in autoupdate manually to point the right link.